class-instant-articles-post.php

```diff
@@ -690,7 +690,9 @@ public function to_instant_article() {
 			Video::setDefaultCommentEnabled( $settings_publishing[ 'comments_on_media' ] );
 		}
 
-		$transformer->transformString( $this->instant_article, $this->get_the_content(), get_option( 'blog_charset' ) );
+		if ( '' != $this->get_the_content() ) {
```

Comment:
Is this happening to you?
I'd change this, so we call only once the `get_the_content()` method here.
Can you do it like this:
```
$the_content = $this->get_the_content();
if ( '' != $the_content ) {
   $transformer->transformString...
}
```